src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +8,48 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val listOfGreetings: MutableList<String> = mutableListOf()
+
+        val englishImplementation = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Hello world"
+            }
+
+            override fun greetSomeone(someone: String): String {
+                return "Hello $someone"
+            }
+        }
+
+        val frenchImplementation = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Salut tout le monde"
+            }
+
+            override fun greetSomeone(someone: String): String {
+                return "Salut $someone"
+            }
+        }
+
+        val spanishImplementation = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Hola, mundo"
+            }
+
+            override fun greetSomeone(someone: String): String {
+                return "Hola, $someone"
+            }
+        }
+
+        listOfGreetings.add(englishImplementation.greet())
```

Comment:
Try to find more flexible solution.
For example, what if there would be a 100 greetings...